eras/conway/impl/src/Cardano/Ledger/Conway/Tx.hs

```diff
@@ -66,6 +72,20 @@ instance Crypto c => EraTx (ConwayEra c) where
       <*> pure valid
       <*> pure (fmap upgradeTxAuxData aux)
 
+getConwayMinFeeTx ::
+  ( EraTx era
+  , AlonzoEraTxWits era
+  , ConwayEraPParams era
+  ) =>
+  PParams era ->
+  Tx era ->
+  Int ->
+  Coin
+getConwayMinFeeTx pp tx refScriptsSize =
+  alonzoMinFeeTx pp tx <+> refScriptsSize <×> refScriptFee
```

Comment:
Good point, and not even hlint complains about it,

Reply:
> not even hlint complains about it

I think that is because those are non-standard operators that are defined in another package (instead of this module) and hlint is not aware of their infix settings. So it can't know if those parenthesis are redundant or not. 

Which is sort of the same reason why I suggested those parenthesis. It was also because those operators are non-standard and the reader might not know the precedence.